oecophylla/taxonomy/taxonomy.rule

```diff
@@ -78,16 +78,18 @@ rule metaphlan2:
 
 rule taxonomy_kraken:
     """
-    Runs Kraken using general defaults.
+    Runs Kraken with Bracken using general defaults.
     """
     input:
         forward = qc_dir + "{sample}/filtered/{sample}.R1.trimmed.filtered.fastq.gz",
         reverse = qc_dir + "{sample}/filtered/{sample}.R2.trimmed.filtered.fastq.gz"
     output:
         map = taxonomy_dir + "{sample}/kraken/{sample}_map.txt",
-        report = taxonomy_dir + "{sample}/kraken/{sample}_report.txt"
+        report = taxonomy_dir + "{sample}/kraken/{sample}_report.txt",
```

Comment:
Do we want to save these Kraken outputs if we're doing kraken with bracken?

I'd prefer to have: 
- 1 rule for kraken-only, which preserves the kraken outputs (in case you need them for some other purpose, seems like it could be useful)
- 1 rule for kraken-bracken, which just outputs the bracken.

Mostly because the kraken outputs are so big. What do you think?

Reply:
Makes sense to me. @sjanssen2 What do you think?

Reply:
Changes made. Tested and working. @tanaes